Code/GraphMol/RGroupDecomposition/testRGroupDecomp.cpp

```diff
@@ -1484,39 +1484,38 @@ N[*:1]
 O[*:1]
 S[*:1]
 C(N[*:1])[*:2]
+P[*:1]
```

Comment:
These "swapped" sides.  This test is pretty fragile and probably should be redone as it doesn't matter how R1 and R2 are aligned.